sql/catalyst/src/main/scala/org/apache/spark/sql/connector/catalog/CatalogManager.scala

```diff
@@ -146,5 +146,5 @@ class CatalogManager(
 }
 
 private[sql] object CatalogManager {
-  val SESSION_CATALOG_NAME: String = "session"
+  val SESSION_CATALOG_NAME: String = "spark_catalog"
```

Comment:
As we see, this cause a test case failure. Could you describe why we change this PR? This is not a documentation change.

Reply:
The config name/document includes the catalog name, so I put them together.

The new name was discussed in a DS v2 community meeting. I'm proposing it in this PR to get more visibility.